File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let unique = 0;
    for(let i=1;i<=nums.length;i++){
        if(nums[i] == nums[i-1])
            continue
        else{
            nums[unique] = nums[i-1];
            unique++;
        }
    }
    return unique
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        k = 0
        
        while i < len(nums):
            if nums[k] != nums[i]:
                k += 1
                nums[k] = nums[i]
            i += 1
        
        return k + 1